packages/kit/src/core/adapt/prerender.js

```diff
@@ -45,6 +45,32 @@ function get_srcset_urls(attrs) {
 	return results;
 }
 
+/** @typedef {((status: number, path: string, parent: string | null, verb: string) => void | never)} InternalErrorHandler */
```

Comment:
Can we call this `PrerenderErrorHandler` or `ErrorHandler` instead? It's not just internal in my mind since the API is exposed for the user to provide it

Reply:
Oh, actually, I see `PrerenderErrorHandler` already defined in `types/config`. Can we just reuse that existing type definition?

Reply:
Actually, it is a slightly different type. The one defined in config is the public version, which gathers all the arguments into a single object, instead of four separate parameters. I made the distinction so that it matched the aesthetic established for the `load` function.

I’ll just make all references to an error-like function in this file just use the single argument version.